tests/software_tests/database/data_record/test_abstract_data_record.py

```diff
@@ -1,9 +1,9 @@
 import pytest
 from mock import Mock, patch
 
-from uds.database.abstract_data_record import AbstractDataRecord
+from uds.database.data_record import AbstractDataRecord
```

Comment:
Let's import directly from the module `uds.database.data_record.abstract_data_record`